### DISCUSSION TITLE
Смена часового пояса

### DISCUSSION THREAD

**Discussion by kilroy98** [shvchk/padavan-builder-workflow#26] [Вопросы и ответы] [answered]
Возможно ли как-то сменить часовой пояс? Сейчас, как я понял, используется стандартный UTC+0.

**Comment by shvchk** [comment]
В самой прошивке, или на сервере сборщика, чтобы в названии архива с прошивкой время было в этом часовом поясе?

**Reply by kilroy98** [reply]
Я так понимаю что на сервере, потому что и в самой прошивке время сборки отображается в этом же часовом поясе.

**Reply by shvchk** [reply]
@kilroy98 так ведь в прошивке часовой пояс настраивается в разделе Администрирование
![image](https://github.com/shvchk/padavan-builder-workflow/assets/872945/1a409156-a04e-4ad7-9ba0-e53826635e39)

**Reply by kilroy98** [reply]
@shvchk но он ведь не влияет на то, о чём я говорю. Я про часовой пояс в названии архива и времени сборки прошивки
![Screenshot_1](https://github.com/shvchk/padavan-builder-workflow/assets/13346898/0f9266ea-223c-45ec-ab9c-e2f12d485418)

**Reply by shvchk** [reply, marked as answer]
@kilroy98 Мне кажется, достаточно изменить часовой пояс сервера сборки, добавив следующие команды в [`pre-build.sh`](https://github.com/shvchk/padavan-builder-workflow/blob/main/README.ru.md#дополнительные-возможности) (на примере часового пояса `Europe/Kyiv`, все зоны есть в [вики](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List)):

```sh
ln -snf /usr/share/zoneinfo/Europe/Kyiv /etc/localtime
apt update
DEBIAN_FRONTEND=noninteractive apt install tzdata -y
```

Если время сборки в самой прошивке указывается по часовому поясу сервера, то должно работать.

**Reply by kilroy98** [reply]
@shvchk спасибо. Как проверю - отпишусь.

**Reply by kilroy98** [reply]
@shvchk да, так работает. Спасибо за помощь!
